open-budget-il/main.py

```diff
@@ -677,22 +680,23 @@ def get_query(self):
         index = search.Index(name="OpenBudget")
         results = []
         try:
-            searchQueryStringList = [queryString]
```

Comment:
the reason is that if the handler has more fine grained search filters we must add the type check to allow the filtering to work.

when q=מדינה&year=2014:
(מדינה AND type=BudgetLine AND year=2014) OR (מדינה AND type=Entity)
And not:
מדינה OR (מדינה AND type=BudgetLine AND year=2014) OR (מדינה AND type=Entity)